target/linux/ipq40xx/image/generic.mk

```diff
@@ -752,6 +753,23 @@ define Device/qcom_ap-dk04.1-c1
 endef
 TARGET_DEVICES += qcom_ap-dk04.1-c1
 
+define Device/tp-link_deco-m5-eu-v3.0
+	$(call Device/FitImage)
+	DEVICE_VENDOR := TP-Link
+	DEVICE_MODEL := Deco-M5-EU
```

Comment:
EU is part of variant, and probably there should be a space before the M5 (at least it is in your PR title) ...